lib/routes/index.js

```diff
@@ -5,14 +5,64 @@
  */
 const express = require('express');
 const router = new express.Router();
+const passport = require('passport');
+const loginMiddleware = require('../auth/login');
+
 
 /*
 Home Page Router
 @function
 @name /
 */
-router.get('/', (req, res) => {
-  res.render('index', { pageTitle: 'Punchcard Home' });
+router.get('/',
+  loginMiddleware(),
```

Comment:
Does putting it just on `/` cover _all_ routes or _just_ `/`? Like if I wanted to go to `/content` would that be covered by the login middleware?